tests/daysOfTheWeek.js

```diff
@@ -17,3 +18,15 @@ tap.test('getDayName', (getDayNameTest) => {
   getDayNameTest.equal(target.getDayName(validDateObject), 'Friday', 'get expected day name when using a Date object');
   getDayNameTest.end();
 });
+
+tap.test('next function', (nextTests) => {
```

Comment:
Great, there's a section of tests for this new function!  The names should be changed to be more meaningful, as with the `next` function.